src/getQuizQuestions/getQuizQuestions.tsx

```diff
@@ -0,0 +1,82 @@
+// 1. Where is Mars in the solar system?
```

Comment:
Pleaser can you remove these comments - it's not necessary to keep them there now the answers have been hardcoded